src/plugins/kibana/server/lib/case_conversion.js

```diff
@@ -0,0 +1,15 @@
+const _ = require('lodash');
+
+module.exports = {
+  convertToSnakeCase: function (object) {
+    return _.mapKeys(object, (value, key) => {
+      return _.snakeCase(key);
+    });
+  },
+
+  convertToCamelCase: function (object) {
```

Comment:
I'm tempted to say that in order for this to be it's own helper it should be deep. I would just put it into the single file that uses it and call it something like `shallowKeysToCamelCase()`